### PR TITLE
Add kubectl-klock

### PR DESCRIPTION
**Pull request by jilleJr**
Adding new plugin: `kubectl klock`

https://github.com/jilleJr/kubectl-klock

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/klock.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    - from: "./klock"
+      to: "."
+    - from: LICENSE
+      to: "."
+    bin: "klock"
+
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
```
Comment (ahmetb) [inline plugins/klock.yaml:42]: highly recommend adding the darwin/arm64 (non-blocking comment)

File: plugins/klock.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: klock
+spec:
+  version: v0.1.1
+  homepage: https://github.com/jilleJr/kubectl-klock
```
Comment (ahmetb) [inline plugins/klock.yaml:11]: highly recommend importing all auth plugins, it seems you're only importing GKE (non-blocking comment)

also recommend adding some examples to --help message of the plugin executable  (non-blocking comment)

**Comment by ahmetb** [comment]
Since I don't have any comments about the manifest and the plugin itself, accepting and you can work on requested changes in a new version.
/lgtm
/approve
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.
Thanks!

**Comment by jilleJr** [comment]
Thanks for the comments!
Will look into those ideas

**Pull request merged**